charts/values.yaml

```diff
@@ -29,3 +29,15 @@ influxdb:
 redis:
   auth:
     enabled: false
+
+kafkaTopic:
```

Comment:
Tror der skal en yderligere selector på for at ramme operatoren?
Og er du sikker på det er operatoren du vil ramme, og ikke selve Kafka clusterets definition? Det ligger i /kafka nu

Reply:
Godt spottet. Jeg har lavet en ny fil i `/kafka` som har de rigtige CRD i stedet. Det burde virke. Men jeg tænker der ikke er andet for end at prøve, haha. Extensive research har vist at det her er det bedste bud.

Reply:
Du må gerne merge. Så ser vi

Reply:
LGTM!